### DISCUSSION TITLE
React + Vite 프로젝트에서 Tailwind CSS 설정하기

### DISCUSSION THREAD

**Discussion by giscus** [padosum/blog#67] [General]
# React + Vite 프로젝트에서 Tailwind CSS 설정하기

Goal React, Vite 로 구성된 프로젝트에서 Tailwind CSS를 사용하는 방법을 알아본다. Tailwind CSS는 이름은 많이 들어봤는데 무엇인지 몰랐다. 첫인상은 Bootstrap이 떠올랐는데 왜 인기가 많은 것일까? 궁금했다. 사…

https://www.padosum.dev/wiki/Setup-Tailwind-CSS-in-React-with-Vite/

**Comment by Earlssu** [comment]
좋은 정보글 감사합니다 🙇🏻‍♂️

블로그 글에서 vite 기반 tailwind 설치에 관해서 쓰려고 하는데, 최연정님의 위키를 언급해도 될까요?

**Reply by padosum** [reply]
얼마든지요! 😄
